react_app/src/apiStuff.env

```diff
@@ -0,0 +1,3 @@
+apiKey = 'af103c190cd36ff3f3fb1e0c135a2ee1'
```

Comment:
Seems like you have the API key in a few places please only put it in one place and reference it from there